conjur/api/api.py

```diff
@@ -20,13 +20,11 @@
 # pylint: disable=too-many-instance-attributes
 from conjur.resource import Resource
 
```

Comment:
why delete all the new lines? is it done by auto-ident?

Reply:
By Mistake, when i removed my checking function
Fixed.